README.md

```diff
@@ -1,8 +1,64 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Minh Phuong Ly . Jarvis Consulting
 
+Being impassioned in exploring and experiencing new technologies, I have knowledge and experience in operating systems, computer networking and software programming. Continuously challenging things, improving and collecting technology knowledge is my favorite so far.
+
+## Skills
+
+**Proficient:** Python, Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git/Github
+
+**Competent:** GitFlow, Flask, JavaScript ES6/JQuery, HTML/CSS, DynamoDB/NoSQL
+
+**Familiar:** S3, EC2, Bootstrap, React.Js, Docker/VMWare, Node.Js, Socket.io, C++
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MinhPhuongly.git](https://github.com/jarviscanada/jarvis_data_eng_MinhPhuongly.git)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MinhPhuongly.git/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_MinhPhuongly/tree/master/linux_sql)]: Implemented a cluster monitoring system which helps to automatically collect hardware specifications from Centos servers every minutes using Crontab and Bash scripts. The data then will be stored in PostgresSQL database provisioned using Docker for later querying to answering some business questions such as the amount of memory used of each server within 5 minutes interval and detect server failures. The PostgresSQL container is also can be start/stop/create by a bash script. We used Git as a version control system to manage the code and followed GitFlow branching strategy.
```

Comment:
**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MinhPhuongly.git/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_MinhPhuongly/tree/master/linux_sql)]: Implemented a cluster monitoring system which helps to automatically collect hardware specifications from Centos servers every minute using Crontab and Bash scripts. The data then will be stored in PostgresSQL database provisioned using Docker for later querying to answering some business questions such as the amount of memory used of each server within 5 minutes interval and detect server failures. The PostgresSQL container is also can controlled by a bash script. We used Git as a version control system to manage the code and followed GitFlow branching strategy.